pkg/reconciler/githubsource/webhook_client.go

```diff
@@ -91,7 +86,12 @@ func (client gitHubWebhookClient) DeleteWithGitHubBaseURL(ctx context.Context, o
 
 	ghClient := client.createGitHubClient(ctx, options)
 	if alternateGitHubAPIURL != "" {
-		ghClient.BaseURL, _ = url.Parse(alternateGitHubAPIURL)
+		baseURL, err := url.Parse(alternateGitHubAPIURL)
+		if err != nil {
+			logger.Infof("Failed to delete webhook - Error occured parsing githubAPIURL %s, error was %v", alternateGitHubAPIURL, err)
+			return fmt.Errorf("Error occured parsing githubAPIURL: %v", err)
```

Comment:
Lowercase 'Error'.

See https://github.com/golang/go/wiki/Errors: "Error strings should not start with a capital letter because they'll often be prefixed before printing"

Reply:
Corrected